### DISCUSSION TITLE
请教个问题，把gotox 放到路由器上运行

### DISCUSSION THREAD

**Discussion by nbzdwss78** [SeaHOH/GotoX#23] [一般]

**Comment by SeaHOH** [comment]
不一定非要做成 egg 包，这是方便我发布 Windows 版本做的。
如果有源里面有依赖包，直接安装就行，如果没有肯定就用不了。
什么源有 openwrt CPU 的版本我也不知道，只能你自己找了。
抱歉，我无法帮助你。

**Comment by SeaHOH** [comment]
应该是一种东西吧，一般写成 pyOpenSSL 而已。你可以进 python-openssl 目录确认一下，是否有 `OpenSSL` 这个包名。
既然安装了 python-openssl ，看看安装日志是否有安装其依赖的 gevent，照理是一并安装了。
你说的不行，那么 GotoX 有什么错误吗？

**Comment by SeaHOH** [comment]
我现在的网络很简单，没条件测试，等以后有机会再测试。估计要过几年搬新房的时候。
如果你实在想用，只能自己折腾了。

python-openssl 目录确认没，是否有 OpenSSL 这个包名？

**Comment by SeaHOH** [comment]
原来如此，那就没法了，我这个用的是 pyOpenSSL。

我搜不到 python-openssl 的官网，这是某个系统的子项目吗？
能不能给个源码地址之类的，目录中应该有相关信息。

**Comment by SeaHOH** [comment]
谢谢 :smile: 
我已经看过了，都是 Python 2.7 的，而且版本很旧，所以 XX-Net 也用不了。
除非有人专门做一个，估计没戏。

**Comment by jzp820927** [comment]
我记得 mini 那边不是已经有人搞定  openwrt 的依赖了麽
